app/presenters/special_announcement_presenter.rb

```diff
@@ -12,20 +12,18 @@ def payload
 private
 
   def special_announcement_properties
-    schema_specific_links = content_item.dig("details", "special_announcement_schema") || {}
-
     {
       "@type" => "SpecialAnnouncement",
-      "category" => schema_specific_links["category"],
+      "category" => "https://www.wikidata.org/wiki/Q81068910",
       "datePosted" => content_item["public_updated_at"],
-      "diseasePreventionInfo" => schema_specific_links["disease_prevention_info_url"],
-      "diseaseSpreadStatistics" => schema_specific_links["disease_spread_statistics_url"],
-      "gettingTestedInfo" => schema_specific_links["getting_tested_info_url"],
-      "newsUpdatesAndGuidelines" => schema_specific_links["news_updates_and_guidelines_url"],
-      "publicTransportClosuresInfo" => schema_specific_links["public_transport_closures_info_url"],
-      "quarantineGuidelines" => schema_specific_links["quarantine_guidelines_url"],
-      "schoolClosuresInfo" => schema_specific_links["school_closures_info_url"],
-      "travelBans" => schema_specific_links["travel_bans_url"],
+      "diseasePreventionInfo" => "https://www.gov.uk/coronavirus",
```

Comment:
I think this should be: "https://www.nhs.uk/conditions/coronavirus-covid-19/"

From: https://github.com/alphagov/govuk-coronavirus-content/blob/master/content/coronavirus_landing_page.yml#L48

Reply:
Thanks - great spot @leenagupte fixed up